modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach errors
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node.
+
+Mounting on a new node is not possible, however, because the failed node was unable to properly unmount the attached volume.
+
+A multi-attach error is reported, such as the one shown in the following example:
+
+[source,terminal]
+.Example output
+--
+Unable to attach or mount volumes: unmounted volumes=[sso-mysql-pvol], unattached volumes=[sso-mysql-pvol default-token-x4rzc]: timed out waiting for the condition
+Multi-Attach error for volume "pvc-8837384d-69d7-40b2-b2e6-5df86943eef9" Volume is already used by pod(s) sso-mysql-1-ns6b4
+--
+
+.Solution
```

Comment:
If the solution is that they can choose either of these options to resolve the issue, I'd suggest to say so with a lead-in sentence.

Reply:
Removed `.Solution` and added an intro sentence.